src/browser-driver-manager.js

```
@@ -1,259 +1,124 @@
-        chalk,
-        verbose,
-        `Received response of ${data['dist-tags'].latest}`
-      );
-
-      if (version <= chromedriverVersion) {
-        await installChromeDriver(version, verbose);
-        console.log(`Successfully installed ChromeDriver ${version}`);
-
-        return;
-      } else {
-        console.log(
-          `ChromeDriver ${version} not found. Retrying with ChromeDriver ${
-            version - 1
-          }`
-        );
-
-        if (version - 1 <= chromedriverVersion) {
-          await installChromeDriver(version - 1, verbose);
-          console.log(`Successfully installed ChromeDriver ${version - 1}`);
-
-          return;
-        } else {
-          return error(
-            chalk,
-            'Unable to get ChromeDriver version; Something went wrong'
-          );
-        }
-      }
-    }
-
-    return;
-  }
-  // output version installed from npm
-  else if (['version', 'which'].includes(userArgs[0]) && chromedriver) {
-    try {
-      const chromedriverPkg = require('chromedriver');
+  return installedBrowser;
+}
 
-      if (userArgs[0] === 'version') {
-        console.log(chromedriverPkg.version);
-      } else {
-        console.log(chromedriverPkg.path);
-      }
-
-      return;
-    } catch (err) {
-      return error(chalk, 'chromedriver is not installed');
-    }
-  }
+async function setEnv({ chromePath, chromedriverPath }) {
+  console.log('Setting env CHROME/CHROMEDRIVER_TEST_PATH');
 
   try {
-    await runBashScript({
-      args: userArgs
-    });
-  } catch (err) {
-    return;
+    await fsPromises.writeFile(
+      path.resolve(BDM_CACHE_DIR, '.env'),
+      `CHROME_TEST_PATH="${chromePath}"\nCHROMEDRIVER_TEST_PATH="${chromedriverPath}"`
+    );
+    console.log(
+      'CHROME_TEST_PATH is set in ',
+      chromePath,
+      '\nCHROMEDRIVER_TEST_PATH is set in ',
+      chromedriverPath
+    );
+  } catch (e) {
+    console.error('Error setting CHROME/CHROMEDRIVER_TEST_PATH', e);
```

`setEnv` should throw the error so node treats this as an error with an exit code (otherwise this just errors but the process completes successfully).